submissions/adeolaFafemi/dom/productCost.js

```diff
@@ -0,0 +1,52 @@
+// selecting elements
+var productPrice = document.getElementById('productprice');
+var shippingCost= document.getElementById('shippingcost');
+var totalCost = document.getElementById('totalcost');
+var input = document.getElementById('quantity');
+var errorInput = document.getElementById('error-input');
+var buyNow = document.getElementById('submit')
+var numExp = /^[0-9]+$/;
+var price = productPrice.innerText
+//functions
+input.addEventListener('input',quantitychanged)
+function quantitychanged(event){
+    var input = event.target
+    if(isNaN(input.value)|| (input.value<=0)|| !numExp.test(input.value)){
+        errorInput.innerText = "Please enter a valid quantity"
+        buyNow.disabled = true;
+    } else{
+        errorInput.innerText = "";
+        buyNow.disabled = false;
+    }
+    updatePrice()
+    updateShipping()
+    updateTotal()
+}
+function updatePrice(){
+        var quantity = input.value;
+        var total = (price*quantity); 
+        productPrice.innerText = total;
+    }
+function updateShipping(){
+    var prodprice = productPrice.innerText;
+    if (prodprice < 1000){
```

Comment:
This block code is confusing. i.e line `32 - 43`. Are you wrapping the `else if` statements inside the `if` block? If yes, that is wrong. If no, kindly format your code to look well align